python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
```

Comment:
I don't think complexity matters here.  What matters is that this file is an implementation detail; it should not be imported anywhere else, and all of its contents should appear to outsiders as if they were defined in this module (or one that imports it).